packages/reports/addon/components/filter-values/date.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{filter-values/date
+ *       filter=filter
+ *       request=request
+ *       onUpdateFilter=(action 'update')
+ *   }}
+ */
+import Component from '@ember/component';
+import Moment from 'moment';
+import layout from '../../templates/components/filter-values/date';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['date'],
+
+  /**
+   * @property {Moment} selectedDate - local moment set by date picker
+   */
+  selectedDate: null,
```

Comment:
`_selectedDate`

Reply:
Do we need to intialize this to `filter.values.firstObject`?

Reply:
Should it be considered private if its passed down to a child component?